hlxsites/merative2
# fix(product-landing): Fix various blocks and section used within MDX landing page

## Issue

Fixes - https://jira.sdlc.merative.com/browse/MERATIVE-822

## Description
- This PR fixes various blocks and sections used within the MDX landing page (`/clinical-decision-support`) version within Franklin.

**New**

-  Stat with Illustration - https://fix-822-product-landing-issues--merative2--proeung.hlx.page/block-library/blocks/stat-with-illustration

![Screenshot 2023-08-31 at 12 23 52 PM](https://github.com/hlxsites/merative2/assets/1815714/e53c19a7-d9aa-40a6-a76b-a52b6e94ade3)

**Changed**

- The following are blocks that have been changed:
  - Linkable Cards - https://fix-822-product-landing-issues--merative2--proeung.hlx.page/block-library/blocks/linkable-cards
  - Content Intro - https://fix-822-product-landing-issues--merative2--proeung.hlx.page/block-library/blocks/content-intro
  - Accordion - https://fix-822-product-landing-issues--merative2--proeung.hlx.page/block-library/blocks/accordion

![Screenshot 2023-08-31 at 12 23 43 PM](https://github.com/hlxsites/merative2/assets/1815714/cb3a338b-2b0b-4b9b-bd78-fc8f5f354ff1)
![Screenshot 2023-08-31 at 12 23 59 PM](https://github.com/hlxsites/merative2/assets/1815714/5ba736a4-ecd4-417e-9022-a4e02f4a0512)
![Screenshot 2023-08-31 at 12 24 08 PM](https://github.com/hlxsites/merative2/assets/1815714/f534330a-04be-4e67-9cc0-964a818fe200)

**Removed**

- Removed the `Content Band Checkmark` block and merged it to the `Content Intro` block instead. This change helps consolidate code (CSS and JS), removes duplications, and creates consistency. FYI. @nimithshetty17 
  - remove `content-band-checkmarks/content-band-checkmarks.js`
  - remove `content-band-checkmarks/content-band-checkmarks.css`

## Design Specs

- Figma Link - https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=1459%3A18597&mode=design&t=b59iXh6I3cYTbZkE-1

## Test URLs
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/clinical-decision-support
- After (Changes from this PR): https://fix-822-product-landing-issues--merative2--proeung.hlx.page/clinical-decision-support
  
## Testing Instruction

- Visit the version of the MDX landing in Franklin (https://fix-822-product-landing-issues--merative2--proeung.hlx.page/clinical-decision-support)
- Ensure that the patterns within this page match the designs.
